From 76fb1967029f4fc4a3712bea2687cd0af2dcf344 Mon Sep 17 00:00:00 2001
From: Juan Nunez-Iglesias <jni@fastmail.com>
Date: Fri, 19 Jan 2024 00:12:49 +0100
Subject: [PATCH] Update working groups leads (#327)

# References and relevant issues

Closes #324

See also [this Zulip thread](https://napari.zulipchat.com/#narrow/stream/298358-working-group-documentation/topic/update-docs-working-group-contacts/near/416509330)

# Description

Our working groups docs are woefully out of date. I've updated to
reflect the current makeup of the groups.
---
 docs/community/working_groups.md | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/docs/community/working_groups.md b/docs/community/working_groups.md
index 0daacdd4..025bc4c8 100644
--- a/docs/community/working_groups.md
+++ b/docs/community/working_groups.md
@@ -18,9 +18,9 @@ We believe working groups are a helpful way to move forward work on particular t
 
 ## Existing working groups
 
-- Bundled Application (Co-Leads: [Gonzalo Peña-Castellanos](https://github.com/goanpeca) and [Ziyang Liu](https://github.com/potating-potato))
-- Architecture (Co-Leads: [Juan Nunez-Iglesias](https://github.com/jni) and [Andy Sweet](https://github.com/andy-sweet))
-- Documentation (Co-Leads: [Genevieve Buckley](https://github.com/GenevieveBuckley) and [Justin Kiggins](https://github.com/neuromusic))
+- Bundled Application (Co-Leads: [Jaime Rodríguez-Guerra](https://github.com/jaimergp) and [Gonzalo Peña-Castellanos](https://github.com/goanpeca))
+- Documentation (Co-Leads: [Peter Sobolewski](https://github.com/psobolewskiPhD) and [Melissa Weber Mendonça](https://github.com/melissawm/))
+- Graphs layer (Co-Leads: [Draga Doncila Pop](https://github.com/DragaDoncila) and [Jordão Bragantini](https://github.com/JoOkuma))
 
 ## How to join an existing working group
 
